src/components/Layout/Header/GlobalSearch/GlobalSearchButton.tsx

```diff
@@ -79,9 +94,34 @@ export const GlobalSeachButton = () => {
     }
   })
 
+  useEffect(() => {
+    if (!searchQuery?.length) return
+
+    setSendResults([])
+    ;(async () => {
+      const parsed = await parseAddressInput({ value: searchQuery })
```

Comment:
not sure how heavy this call is, but wondering if we debounce our search inputs at all to avoid running this on every character entered in search input

Reply:
Yeah I originally debounced it but we already debounce the search in, see 082c95051e4071bf0519ecce06ea2ed42a333a2f